app/Filament/Server/Resources/ActivityResource.php

```diff
@@ -30,8 +30,7 @@ public static function getEloquentQuery(): Builder
         /** @var Server $server */
         $server = Filament::getTenant();
 
-        return $server->activity()
-            ->getQuery()
+        return ActivityLog::whereHas('subjects', fn (Builder $query) => $query->where('subject_id', $server->id))
```

Comment:
```suggestion
        return QueryBuilder::for($server->activity())
```
https://github.com/pelican-dev/panel/blob/b44411208573132c8be73dc2959ac90130442ddb/app/Http/Controllers/Api/Client/Servers/ActivityLogController.php#L34

Reply:
That's effectively the same as `$server->activity()->getQuery()`